main.rb

```diff
@@ -1,11 +1,121 @@
-require_relative './person'
-require_relative './decorator'
+require_relative './app'
 
-person = Person.new(22, 'maximilianus')
-puts person.correct_name
+def list_of_options
+  puts 'Please choose an option by entering a number:'
+  puts '1 - List all books'
+  puts '2 - List all people'
+  puts '3 - Create a person'
+  puts '4 - Create a book'
+  puts '5 - Create a rental'
+  puts '6 - List all rentals for a given id'
+  puts '7 - exit'
+end
 
-capitalized_person = CapitalizeDecorator.new(person)
-p capitalized_person.correct_name
+def actions(option, app)
+  case option
+  when 1
+    list_books(app)
+  when 2
+    list_people(app)
+  when 3
+    create_person(app)
+  when 4
+    create_book(app)
+  when 5
+    create_rental(app)
+  when 6
+    list_rentals_by_id(app)
+  end
+end
 
-capitalized_trimmed_person = TrimmerDecorator.new(capitalized_person)
-p capitalized_trimmed_person.correct_name
+def list_books(app)
+  app.list_books
+end
+
+def list_people(app)
+  app.list_people
+end
+
+def create_person(app)
+  puts 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
+  selection = gets.chomp.to_i
+
+  if [1, 2].include?(selection)
+    puts 'Age: '
+    age = gets.chomp.to_i
+    puts 'Name: '
+    name = gets.chomp
+
+    case selection
+    when 1
+      puts 'Has parent permission? [Y/N]: '
+      parent_permission = gets.chomp
+      app.create_student(age, name, parent_permission)
+    when 2
+      puts 'Specialization: '
+      specialization = gets.chomp
+      app.create_teacher(age, name, specialization)
+    end
+  else
+    puts 'Wrong Selection, returning to menu'
+  end
+end
+
+def create_book(app)
+  puts 'Title: '
+  title = gets.chomp
+  puts 'Author: '
+  author = gets.chomp
+  app.create_book(title, author)
+end
+
+def create_rental(app)
+  return unless app.list_books_with_index.nil? do
+    # Display options
+    puts 'Select a book from the list by index number: '
+    book_i = gets.chomp.to_i
+    puts ''
+
+    # Display People Options
+    app.list_people_with_index
+    puts 'Select a person from the list by index number(not id): '
+    person_i = gets.chomp.to_i
+    puts ''
+
+    # Date option
+    puts 'Date: '
+    date = gets.chomp
+
+    app.create_rental(date, person_i, book_i)
+  end
+end
```

Comment:
- [ ] Please fix this error instead to create rental a the list of books is the display 
![rental](https://user-images.githubusercontent.com/55596970/188282316-df7632eb-cc19-44dc-b003-4f453730a2f2.png)